### PR TITLE
Improve ReadMe

### PR DESCRIPTION
**Pull request by confused-Techie**
This PR adds the new site features to our ReadMe while removing outdated information.

Additionally adds a section to the readme on how to create new themes, with detailed instructions on what needs to be done.

@kyjus25 if you wouldn't mind adding a suggestion to this one with a good explanation of the Dracula theme. You can use the other's added here as reference, but wanted to ensure whatever message used was accurate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,11 @@
 
 Serves as a visual Web Browser for the [`package-backend`](https://github.com/pulsar-edit/package-backend).
 
-Currently this matches very closely with the design of the original package viewer from upstream Atom.
-
-Although does fully support multiple themes being added on later with a theme switcher.
+While the site is still in it's early stages it has an original design and supports several themes already:
+  * Atom.io Theme - Theme Based off the Original Atom Package Browser
+  * GitHub Dark - A theme Based off GitHub Dark Theme
+  * Dracula - <@kyjus25 Please feel free to write an accurate description>
```
Comment (kyjus25) [inline README.md:8]: "Theme based off the versatile [Dracula](https://draculatheme.com/) theme 🧛"

**Comment by kyjus25** [comment]
Comment added of my suggestion. May drop the emoji so the descriptions are more or less uniform

File: README.md (hunk 1 above)
Comment (confused-Techie) [reply README.md:8]: Love it, I'll add it when I'm back at home, thanks!

Comment (confused-Techie) [inline README.md:8]:
```suggestion
  * Dracula - Theme based off the versatile [Dracula](https://draculatheme.com/) theme🧛
```

Forgot I could just do this.

**Comment by confused-Techie** [comment]
@kyjus25 Look good?

**Comment by kyjus25** [comment]
@confused-Techie Looks good!

**Pull request merged**